src/clj/rems/migrations/add_organization_owner_role.clj

```diff
@@ -0,0 +1,35 @@
+(ns rems.migrations.add-organization-owner-role
+  (:require [hugsql.core :as hugsql]
+            [rems.json :as json]))
+
+;; This migration goes through all api keys defined in api_key table and
+;; adds :organization-owner to all lists of permitted roles that already
+;; include :owner.
```

Comment:
we might not need this migration, it might be enough to mention the new role in the changelog and installations can consider adding it. on the other hand, adding organization-owner can't hurt if somebody already has owner rights

Reply:
True. I kept this migration here for now, as I believe (but am not sure) that we have at least one installation where we supposedly want to have organization-owner in the permitted roles, but it doesn't exist there currently. So this should save the step of editing the database manually in that case, and probably doesn't hurt elsewhere.